docs/source/blogs/release-0-3-0-blog/release-0-3-0-blog.rst

```diff
@@ -0,0 +1,169 @@
+
+
+.. figure:: figsplitdf.png
+   :align: center
+   :width: 800
+   :name: figsplitdf
+
+   The split DataFrame ``split_variants_df``
+
+It should be note that the new splitter is implemented as a separate transformer from the ``normalize_variants`` transformer, as opposed to the previous version's splitter where the splitting could only be done as one of the operation modes of the ``normalize_variants`` transformer using the now-deprecated ``mode`` option.
+
+Please refer to the :ref:`documentation of the split_multiallelics transformer<split_multiallelics>` for complete details on the bahavior of this new transformer.
+
+
+Python and Scala APIs for Glow SQL functions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Excitingly, in this release, Python and Scala APIs were introduced for all Glow SQL functions. Therefore, unlike previous versions where Glow SQL functions could only be used in SQL expressions, they can now be written in Python or Scala as well similar to what can be done for Spark SQL functions. In addition to improved simplicity, this provides enhanced compile-time safety. The SQL functions and their Python and Scala clients are generated from the same source so any new functionality in the future will always appear in all three languages. Please refer to :ref:`pyspark_functions` for more information on Python APIs for these functions.
+
+As an example, the ``normalize_variant`` function which was used within a SQL expression above, can also be written as a Python function as follows:
+
+.. code-block::
+
+  from glow.functions import normalize_variant
+  function_normalized_variants_df = original_variants_df.withColumn( \
+    "normalizationResult", \
+    normalize_variant("contigName", "start", "end", "referenceAllele", "alternateAlleles", "/mnt/dbnucleus/dbgenomics/grch38/data/GRCh38_full_analysis_set_plus_decoy_hla.fa") \
+  )
+
+The same can be done in Scala assuming ``original_variant_df`` is defined in Scala:
+
+.. code-block:: scala
+
+  import io.projectglow.functions.normalize_variant
+  import org.apache.spark.sql.functions.col
+  val function_normalized_variants_df = original_variants_df.withColumn(
+    "normalizationResult",
+    normalize_variant(col("contigName"), col("start"), col("end"), col("referenceAllele"), col("alternateAlleles"), "/mnt/dbnucleus/dbgenomics/grch38/data/GRCh38_full_analysis_set_plus_decoy_hla.fa")
+  )
+
+Parsing of Annotation Fields
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In this release, the VCF reader and pipe transformer received equipment to parse ``ANN`` and ``CSQ`` ``INFO`` fields resulting form annotation tools such as `SnpEff <http://snpeff.sourceforge.net/index.html>`_ and  `VEP <https://www.ensembl.org/info/docs/tools/vep/index.html>`_ tools into a structured format instead of putting them as flat strings in the ``INFO`` field. This makes queries on annotations much simpler to write and much faster. An example of such a query in shown below with its result displayed in :numref:`figcsqdf`:
```

Comment:
The VCF reader and pipe transformer now parse variant annotations from tools such as `SnpEff <http://snpeff.sourceforge.net/index.html>`_ and  `VEP <https://www.ensembl.org/info/docs/tools/vep/index.html>`_. This parse flattens the ``ANN`` and ``CSQ`` ``INFO`` fields, simplifying and accelerating queries on annotations. See :numref:`figcsqdf`: for an example.